catroid/src/main/java/org/catrobat/catroid/utils/FlashUtil.java

```diff
@@ -51,8 +51,13 @@ public static boolean isOn() {
 		return currentFlashValue;
 	}
 
+	public static boolean isPaused() {
+		return paused;
+	}
+
 	public static void pauseFlash() {
-		Log.d(TAG, "pauseFlash");
+		Log.d(TAG, "pauseFlash()");
```

Comment:
just remove the log altogether, as this was clearly added for debugging purposes. 

I think it is just bad practice to use log for debugging instead of Android Studio's debugger. breakpoints ftw! :smile: